sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/WebJobs.Extensions.AuthenticationEvents/ConfigurationManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
+{
+    internal class ConfigurationManager
+    {
+        private readonly string[] SERVICE_IDs = new string[] { "99045fe1-7639-4a75-9d4a-577b6ca3810f" };
+        internal const string OPENID_CONNECTHOST = "https://login.microsoftonline.com";
```

Comment:
What happens if we're running in a sovereign or regional cloud where this is not the AAD authority host?

Reply:
We have provision for that, the idea is that we would be able to manager multiple services... hence the array. When the token is first validated it determines the service, from there, it's cached. We do not have the services for this yet, but once we do, we will update this. As far as the host is concerned, we would need to apply the same logic.

Reply:
This is now done.